aten/src/ATen/core/List.h

```diff
@@ -25,8 +25,9 @@ struct ListImpl final : public c10::intrusive_ptr_target {
 
   explicit ListImpl(list_type list_, optional<TypePtr> elementType_)
   : list(std::move(list_))
-  , elementType(std::move(elementType_))
-  {}
+  , elementType(std::move(elementType_)) {
+    TORCH_INTERNAL_ASSERT(!elementType.has_value() || nullptr != elementType->get(), "Key and value type must not be nullptr");
```

Comment:
change error message?

Reply:
to what?

Reply:
It talks about "key and value" (so looks like from dict), just "elements"?

Reply:
ah didn't catch that. Thanks.